selfdrive/common/messagehelp.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
```

Comment:
Wow, I didn't know about this! We should use it everywhere I think.

https://en.wikipedia.org/wiki/Pragma_once